netcore/System.Private.CoreLib/src/System/Environment.Mono.in

```diff
@@ -1,8 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please no `.in` in csproj

Reply:
`Environment.Mono.in` being processed into `Environment.Mono.cs` by Autoconf is already a thing - I've moved the contents of `Environment.cs` into `Environment.Mono.in` because CoreCLR already has an `Environment.cs`

How to deal with generated files is an ongoing conversation in email with monorepo stakeholders.